flight/camera.py

```diff
@@ -159,31 +158,35 @@ async def odlc_move_to(
                 drone_long: float = position.longitude_deg
                 drone_alt: float = position.relative_altitude_m
 
-                #  accurately checks if location is reached and stops for 15 secs and then moves on.
-                if (
-                    (round(drone_lat, int(6 * fast_param)) == round(latitude, int(6 * fast_param)))
-                    and (
-                        round(drone_long, int(6 * fast_param))
-                        == round(longitude, int(6 * fast_param))
-                    )
-                    and (round(drone_alt, 1) == round(altitude, 1))
-                ):
+                total_distance: float = calculate_distance(
+                    drone_lat, drone_long, drone_alt, latitude, longitude, altitude
+                )
+
+                if total_distance < WAYPOINT_TOLERANCE:  # 6 meters = 19.685 feet.
```

Comment:
WAYPOINT_TOLERANCE is currently 1 meter, yet the comment indicates that its 6 meters. I would prefer a higher tolerance, so we can fly through the waypoints.